scripts/generate_maps.py

```diff
@@ -0,0 +1,177 @@
+import geopandas as gpd
```

Comment:
Please find a linter + autoformatter, and configure it to sort your imports alphabetically. See my message on Zulip.